tests/playwright/src/model/pages/create-machine-page.ts

```diff
@@ -0,0 +1,137 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+
+import { delay } from '/@/utility/wait';
+
+import { BasePage } from './base-page';
+import { ResourcesPage } from './resources-page';
+
+export class CreateMachinePage extends BasePage {
+  readonly heading: Locator;
+  readonly podmanMachineName: Locator;
+  readonly podmanMachineConfiguration: Locator;
+  readonly imagePathBox: Locator;
+  readonly browseImagesButton: Locator;
+  readonly podmanMachineCPUs: Locator;
+  readonly podmanMachineMemory: Locator;
+  readonly podmanMachineDiskSize: Locator;
+  readonly rootPriviledgesCheckbox: Locator;
+  readonly userModeNetworkingCheckbox: Locator;
+  readonly startNowCheckbox: Locator;
+  readonly closeButton: Locator;
+  readonly createMachineButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.heading = this.page.getByRole('heading', { name: 'Create Podman Machine' });
+    this.podmanMachineConfiguration = this.page.getByRole('form', { name: 'Properties Information' });
+    this.podmanMachineName = this.podmanMachineConfiguration.getByRole('textbox', { name: 'Name' });
+    this.imagePathBox = this.podmanMachineConfiguration.getByRole('textbox', { name: 'Image Path (Optional) ' });
+    this.browseImagesButton = this.podmanMachineConfiguration.getByRole('button', {
+      name: 'button-Image Path (Optional)',
+    });
+    this.podmanMachineCPUs = this.podmanMachineConfiguration.getByRole('slider', { name: 'CPU(s)' });
+    this.podmanMachineMemory = this.podmanMachineConfiguration.getByRole('slider', { name: 'Memory' });
+    this.podmanMachineDiskSize = this.podmanMachineConfiguration.getByRole('slider', { name: 'Disk size' });
+    this.rootPriviledgesCheckbox = this.podmanMachineConfiguration.getByRole('checkbox', {
+      name: 'Machine with root privileges',
+    });
+    this.userModeNetworkingCheckbox = this.podmanMachineConfiguration.getByRole('checkbox', {
+      name: 'User mode networking',
+    });
+    this.startNowCheckbox = this.podmanMachineConfiguration.getByRole('checkbox', { name: 'Start the machine now' });
+    this.closeButton = this.page.getByRole('button', { name: 'Close' });
+    this.createMachineButton = this.page.getByRole('button', { name: 'Create' });
+  }
+
+  async createMachine(
+    machineName: string,
+    { isRootful = true, enableUserNet = false, startNow = true, setAsDefault = true },
+  ): Promise<ResourcesPage> {
+    await playExpect(this.podmanMachineConfiguration).toBeVisible();
+    await this.podmanMachineName.fill(machineName);
```

Comment:
```suggestion
    await this.podmanMachineName.clear();
    await this.podmanMachineName.fill(machineName);
```

Usually this field has a default value I think, the textbox should be cleared before entering the desired name.

Reply:
I think that filling it in works as expected, otherwise test would fail. Regardless, clearing it before is good practice.